### PR TITLE
Feat:lesson11

### PR DESCRIPTION
**Pull request by sae-github**
# [Lesson11](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#11)
 It retrieves the data using fetch, processes it as before, and displays it in the browser.
## Specification Description
I made it so that if the status code is not 2xx, it throws an error to response.ok.
I also wrote an if statement to prevent the forEach from happening if the data is undefined.

###  [ Comment ]
I had a hard time with the error handling part.
It was difficult to know how far to go in anticipating and catching errors and unintended actions.
If you have any advice, I would appreciate it.

Please leave a review!

### CODE REVIEW

File: lesson11/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  } catch (e) {
+    const div = document.getElementById('js-wrapper');
+    div.textContent = "エラーが発生しました！";
+    console.error(e);
+  } 
+}
+
+
+async function createLists() {
+  const responseData = await getData();
```
Comment (kenmori) [inline lesson11/index.js]: if response is undefined,what do you do.

Comment (sae-github) [reply lesson11/index.js]: Thank you for your review.

Added a process to display text in the browser if responseData is undefined. 
Please check it.

[Latest commit](167043c)

**Comment by sae-github** [comment]
I made some corrections.

If the status number was not 2xx due to a server error, it threw an error and displayed to the browser that a server error had occurred.
Also, if there is no 'data' in the retrieved data, the browser will display "No appropriate data found".

Please review.

5eedde1ec302a90e6250ac1941fcee4d21162ece

**Comment by sae-github** [comment]
Corrections have been made.
Added trycreate function.
If the JSON data can be retrieved without any problem, 
the createLists function will be executed, but if a problem occurs, an exception will be thrown.

9542697942db218ad02b293d19ae868ed4b8e093
https://codesandbox.io/s/js-lesson11-rxm69

**Comment by kenmori** [comment]
Approve

**Pull request merged**